packages/ibm-products/src/components/Datagrid/utils/DatagridActions.js

```diff
@@ -101,7 +96,7 @@ export const DatagridActions = (datagridState) => {
         {!mobileToolbar ? (
           <>
             {renderFilterPanelButton()}
-            <div style={style}>
+            <div>
```

Comment:
Do we still need these empty `div`s? 🤔 Or are they safe to remove if they were only used for that `style` attribute.